pkg/server/commands/connectedobjects/connected_objects.go

```diff
@@ -149,14 +158,14 @@ type ConnectedObjectsResult struct {
 	ResultStatus ConditionalResultStatus
 }
 
-func (c *ConnectedObjectsCommand) streamedConnectedObjects(
+func (c *ConnectedObjectsQuery) execute(
 	ctx context.Context,
 	req *ConnectedObjectsRequest,
 	resultChan chan<- *ConnectedObjectsResult,
 	foundObjectsMap *sync.Map,
 	foundCount *uint32,
 ) error {
-	ctx, span := tracer.Start(ctx, "streamedConnectedObjects", trace.WithAttributes(
+	ctx, span := tracer.Start(ctx, "execute", trace.WithAttributes(
```

Comment:
Unfortunate we lose the name. `execute` is so generic as a span name it's almost not helpful. I realize it's scoped/namespaced to the package `openfga/pkg/server/commands/connected_objects`, but that requires looking at 2 fields to see what we're dealing with..

Reply:
Renamed to `connectedObjects.execute`